### ISSUE TITLE
Contribute to this repo!

### ISSUE THREAD

**Issue by Atharv-Attri** [Atharv-Attri/HacktoberFest-Projects#11]
Read the readme and start contributing! Please make sure you read the quality section carefully! We have high standards on what we merge!

**Comment by salilbc** [comment]
Are there any specific programs you are looking at?

**Comment by Atharv-Attri** [comment]
@salilbc not really. You choose what you want to make, and as long as it's a quality project, I approve it!

**Comment by yashqwerty1** [comment]
I would like to contribute here. I will send my project here. Please do check and pull it.

**Comment by Atharv-Attri** [comment]
@yashqwerty1 try to make an issue first to confirm it meets the quality standards! I would hate it if your work was for nothing!

**Comment by yashqwerty1** [comment]
Its A GUI Calculator

**Comment by Atharv-Attri** [comment]
@yashqwerty1 please create a new issue. And I think that might be a little too simple unless you add in all the complex math stuff too.

**Comment by yashqwerty1** [comment]
Ok that would be Great thing to add

**Comment by Atharv-Attri** [comment]
Great, create an issue and I'll assign it to you!

**Comment by yashqwerty1** [comment]
Instead I can go for some Games with python for fun Because i never made one. Still You can assign me this.

**Comment by Atharv-Attri** [comment]
Sounds great!

**Comment by yashqwerty1** [comment]
Yes. Can you assign me this task?

**Comment by Atharv-Attri** [comment]
You need to create a new issue first @yashqwerty1

**Comment by yashqwerty1** [comment]
@Atharv-Attri Created it.

**Comment by archit2216** [comment]
can i do a problem involving recursion?

**Comment by Atharv-Attri** [comment]
Sure, if the problem is complex enough! Create an issue with what you have in mind
